### PR TITLE
Added 12 Poland companies

### PR DESCRIPTION
**Pull request by top-corder**

### CODE REVIEW

File: Europe.md [hunk 1]
```diff
@@ -64,4 +64,20 @@
...
+18. https://jooble.org/ (job platform)
+19. https://www.alamedadev.com/join-our (frontend, backend)
+20. https://www.inatlas.com/join-us/ (Angular, React, Vue)
+21. https://theorg.com/org/argilla-io/jobs (Python)
+
+
+# Poland
+1. https://www.boldare.com/career/offers/ (React, Vue)
+2. https://dev.pro/career/ (React)
+3. https://career.orangesoft.co/ (Node.js0)
```
Comment (waitingwittykitty) [inline Europe.md:76]: You should remove 0 at the end of the tech stack

**Review by waitingwittykitty — CHANGES_REQUESTED** [review]
Please check the typo.

**Comment by top-corder** [comment]
I fixed it. PTAL

**Review by startjourney9 — APPROVED** [review]
👍

**Review by waitingwittykitty — APPROVED** [review]

**Pull request merged**